packages/sitecore-jss/src/site/graphql-siteinfo-service.ts

```diff
@@ -35,16 +35,6 @@ export type SiteInfo = {
 };
 
 export type GraphQLSiteInfoServiceConfig = CacheOptions & {
```

Comment:
Since _endpoint_ and _apiKey_ are removed, _clientFactory_ should be a required property of each service.
Currently it's optional